### PR TITLE
Improve docs for Decorators concept

### PR DESCRIPTION
**Pull request by velaco**
Improve docs for Decorators concept

Related issue: #3120

### CODE REVIEW

**Comment by BethanyG** [comment]
Hi @velaco 👋🏽 

Thank you so much for submitting this PR!  I took a quick look, and it generally looks good.  However, `class decorators` are out of scope for this particular concept/concept exercise (_see #3119 for some details_).  Could you please remove references to class decorators from the blurb, introduction, and links? _Many thanks_.

Beyond those corrections, I think this is good to go live.  😄

**Review by BethanyG — CHANGES_REQUESTED** [review]
Please remove the references to class decorators, as noted.

**Comment by velaco** [comment]
Hi @BethanyG !

Yes, you are right, I missed that issue and was actually going to ask if there are plans to expand `about.md` with decorators. Guess you answered the question before I asked it 😄 
Anyway, thanks for the review! I updated the PR just now.

**Review by BethanyG — APPROVED** [review]
Awesome!  Thank you for working on this.  🌟

File: concepts/decorators/.meta/config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for this concept",
-  "authors": ["bethanyg", "cmccandless"],
-  "contributors": []
+  "blurb": "Decorators are higher-order functions that take other functions as arguments to extend/modify their behavior and return the modified functions.",
+  "authors": ["BethanyG", "bobahop", "kotp", "mathstrains21"],
+  "contributors": ["velaco"]
```
Comment (BethanyG) [inline concepts/decorators/.meta/config.json:4]:
```suggestion
  "authors": ["BethanyG", "bobahop", "kotp", "mathstrains21", "velaco"],
  "contributors": []
```

**Pull request merged**